src/components/results/common/results-global-filter.tsx

```diff
@@ -0,0 +1,269 @@
+        },
+        '.Mui-expanded, .Mui-focused, .Mui-focusVisible': {
+            position: 'absolute',
+            width: 'inherit',
+            height: 'inherit',
+            zIndex: 20,
+            background: theme.palette.tabBackground,
+        },
+        '.MuiInputLabel-root': {
+            zIndex: 30,
+            width: 'auto',
+        },
+    }),
+    chipBox: {
+        width: '100%',
+        display: 'flex',
+        flexWrap: 'wrap',
+        padding: '0.5em',
+    },
+    recentBox: (theme: Theme) => ({
+        borderBottom: '1px solid',
+        borderColor: theme.palette.divider,
+    }),
+    recentLabel: (theme: Theme) => ({
+        color: theme.palette.text.secondary,
+        fontSize: 'small',
+        width: '100%',
+        paddingLeft: 1,
+    }),
+    chip: {
+        '&.MuiChip-root': {
+            margin: '4px 2px 4px 2px',
+            padding: '0',
+            color: 'white',
+        },
+        '.MuiChip-deleteIcon:hover': {
+            color: 'white',
+        },
+        '&.Mui-focusVisible': {
+            width: 'unset', // prevents the chip from changing size when selected with the keyboard
+            height: 'unset', // prevents the chip from changing size when selected with the keyboard
+            position: 'relative',
+        },
+    },
+    chipCountry: {
+        '&.MuiChip-root, &.MuiChip-root[aria-selected="true"]': {
+            backgroundColor: '#01579b !important',
+        },
+        '.MuiChip-deleteIcon': {
+            color: '#b2daf1',
+        },
+        '&.MuiChip-root:hover': {
+            backgroundColor: '#0277bd !important',
+        },
+        '&.Mui-focusVisible': {
+            background: '#0288d1 !important',
+        },
+    },
+    chipVoltageLevel: {
+        '&.MuiChip-root, &.MuiChip-root[aria-selected="true"]': {
+            backgroundColor: '#1e88e5 !important',
+        },
+        '& .MuiChip-deleteIcon': {
+            color: '#bbdefb',
+        },
+        '&.MuiChip-root:hover': {
+            backgroundColor: '#1976d2 !important',
+        },
+        '&.Mui-focusVisible': {
+            background: '#2196F3 !important',
+        },
+    },
+};
+
+export interface Filter {
+    label: string;
+    filterType: string;
+    recent?: boolean;
+}
+
+export interface ResultsGlobalFilterProps {
+    onChange: (value: Filter[]) => void;
+    filters: Filter[];
+}
+
+const emptyArray: Filter[] = [];
+
+const ResultsGlobalFilter: FunctionComponent<ResultsGlobalFilterProps> = ({
+    onChange,
+    filters = emptyArray,
+}) => {
+    const intl = useIntl();
+    const { translate } = useLocalizedCountries();
+    const dispatch = useDispatch();
+    const recentGlobalFilters = useSelector(
+        (state: ReduxState) => state.recentGlobalFilters
+    );
+
+    const getOptionLabel = useCallback(
+        (option: Filter) =>
+            option.filterType === 'country'
+                ? translate(option.label)
+                : option.label + ' kV',
```

Comment:
Use an enum for properties to filter ?
enum FilterType {
    COUNTRY =  'country',
    VOLTAGE_LEVEL= 'voltagelevel'

}

Reply:
Done